### DISCUSSION TITLE
[V4] Compatible Lucid version for Adonis 4.1

### DISCUSSION THREAD

**Discussion by joeneldeasis** [adonisjs/core#2466] [Help]
Hi guys what's the latest compatible version of lucid for Adonis 4.1? 
Based on the adonis-api-app it still uses `"@adonisjs/lucid": "^6.1.3"`.

**Comment by xmajzel** [comment]
Hello,

based on versions mentioned in npm [@adonisjs/lucid](https://www.npmjs.com/package/@adonisjs/lucid) last supported version for Adonis 4.1 should be 6.3.0 (marked as legacy).

**Comment by RomainLanz** [comment]
Hey @joeneldeasis! 👋 

Every package has been published for Adonis 4.1 with the `@legacy` tag.

```
npm i @adonisjs/lucid@legacy
```
